gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/process/ProcessStandardSuite.java

```diff
@@ -144,6 +145,7 @@ public class ProcessStandardSuite extends AbstractGremlinSuite {
             AddEdgeTest.Traversals.class,
             AddVertexTest.Traversals.class,
             CoalesceTest.Traversals.class,
+            ConcatTest.Traversals.class,
```

Comment:
i think TinkerPop needs to start pressing on providers to get off of the old java process tests. its time consuming to maintain both these and the gherkin tests as duplicates. not sure if it's good to start this path where all the new string functions that follow will need java and gherkin tests. thoughts?

Reply:
I would agree for string functions there is no need to keep the duality. I will remove it here.